### PR TITLE
fix: add SamarthSaxena10

### LINKED ISSUE #1634: [PROFILE ADDITION] <Samarth Saxena>
### Profile screenshot

screencapture-github-SamarthSaxena10-2023-01-09-23_44_55

### PR DESCRIPTION
**Pull request by SamarthSaxena10**
resolves #1634

### CODE REVIEW

**Review by krshkun — APPROVED** [review]
Looks good to me

File: profiles/SamarthSaxena10.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image":"https://user-images.githubusercontent.com/90990378/211378645-134aba56-6085-4db4-a02b-61d07fc275dd.png",
```
Comment (Panquesito7) [inline profiles/SamarthSaxena10.json:2]: Non-blocking: missing a space here.
```suggestion
  "image": "https://user-images.githubusercontent.com/90990378/211378645-134aba56-6085-4db4-a02b-61d07fc275dd.png",
```

**Review by Panquesito7 — APPROVED** [review]
Overall, it looks good. Thanks! 🎉

**Comment by SamarthSaxena10** [comment]
Updated !

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**